### PR TITLE
ENH: Add FLUID column and simplify column names in inplace_volumes

### LINKED ISSUE #903: Inplace volumes: Add `FLUID` column
We should add a new column in the export of inplace volumes that contains the fluid zone such that we can simplify other columns names, and also reduce the number of them.

I.e. instead of `BULK_OIL`,  `BULK_GAS` we just have `BULK` and set `FLUID` to `OIL` or `GAS`, respectively.

### PR DESCRIPTION
**Pull request by tnatt**
PR to add a FLUID column to the exported inplace_volumes table.
There will be more tests of the table format coming in upcoming PR #900 

Closes #903

### CODE REVIEW

File: src/fmu/dataio/export/rms/inplace_volumes.py [hunk 1]
```diff
@@ -24,7 +25,17 @@
 _logger: Final = null_logger(__name__)
 
 
-_TABLE_INDEX_COLUMNS: Final = ("ZONE", "REGION", "FACIES", "LICENCE")
+_TABLE_INDEX_COLUMNS: Final = ("ZONE", "REGION", "FACIES", "LICENSE")
```
Comment (tnatt) [inline src/fmu/dataio/export/rms/inplace_volumes.py:28]: typo in `LICENSE` fixed

Comment (mferrera) [reply src/fmu/dataio/export/rms/inplace_volumes.py:28]: I think this was not your typo:

https://github.com/equinor/fmu-dataio/blob/61d17d4b54dddcd901a9ea6ff56485a6b3c0d95a/src/fmu/dataio/_definitions.py#L73

We probably want to change this too? Doesn't need to be in your PR

File: tests/data/drogon/tabular/volumes/geogrid.csv [hunk 2]
```diff
@@ -0,0 +1,337 @@
+ZONE,REGION,FACIES,BULK,PORV,HCPV,GIIP,ASSOCIATEDOIL,FLUID,STOIIP,ASSOCIATEDGAS
```
[the remark this answers is not in the document]
Comment (tnatt) [reply tests/data/drogon/tabular/volumes/geogrid.csv:1]: Yes agree, on all above :) I'll make the change! I realized now that I forgot to add the FLUID column as a TABLE_INDES, that will shift that column to the right 👍

File: src/fmu/dataio/export/rms/inplace_volumes.py (hunk 1 above)
Comment (tnatt) [reply src/fmu/dataio/export/rms/inplace_volumes.py:28]: 👍 I'll make an issue of it, and also state that we should add `FLUID` in there

File: tests/data/drogon/tabular/volumes/geogrid.csv (hunk 2 above)
[the remark this answers is not in the document]
Comment (mferrera) [reply tests/data/drogon/tabular/volumes/geogrid.csv]: I'm not sure if you understood by right I did actually mean left, or yourself confused them as I did, but looks good now 😄

[the remark this answers is not in the document]
Comment (tnatt) [reply tests/data/drogon/tabular/volumes/geogrid.csv]: hehe OMG yes meant left 😅 sometimes I consider writing L and R on my hands like I did when I took my drivers license

**Pull request merged**